app/views/layouts/_topnav.html.haml

```diff
@@ -54,13 +54,10 @@
             - s.item do
               = link_to(t('layout.header.submit_feedback'), feedback_path(location: encode_param(request.url)), id: "submitFeedbackMenuItem", class: "pop_window")
           - d.with_section do |s|
-            - s.header do
-              = t('layout.header.documentation')
             - s.item do
-              = link_to(t('layout.header.help'), "#{$WIKI_HELP_PAGE}", target: "_blank")
+              = link_to(t('layout.header.documentation'), "https://ontoportal.github.io/documentation/", target: "_blank")
```

Comment:
this need to use the configuration, not been hard coded

Reply:
done